### PR TITLE
Added Pull request templated based on https://github.com/axolo-co/pul…

### PR DESCRIPTION
**Pull request by ninjapanzer**
# Description

Noticed there was not a PR template in this repo so here is one I pulled from https://github.com/axolo-co/pull_request_template

Fixes # N/A

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

N/A

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+# Checklist:
+
+- [ ] My code follows the style guidelines of this project
```
Comment (athix) [inline .github/pull_request_template.md:25]: I don't think we have a style guide yet, actually.

Is there anything similar in nature to [Ruby's Style Guide](https://rubystyle.guide/) but for JavaScript / NodeJS?

**Review by athix — APPROVED** [review]
I think there are some minor tweaks that we can do to improve the PR experience even further, but this is a great first step in the right direction.

We'll roll with this for now, and I'll add to this as we go.

File: .github/pull_request_template.md (hunk 1 above)
Comment (ninjapanzer) [reply .github/pull_request_template.md:25]: Yea, I am 💯 sure we can find something to append to the eslint configuration as that enforces the codestyle for the mostpart. I plan to open an issue adding lint to gh-actions and start a conversation about code style options.

Comment (ninjapanzer) [reply .github/pull_request_template.md:25]: @garyanewsome have any opinions about eslint configuration and a solid codestyle? ☝️

**Pull request merged**